app/controllers/active_storage/base_controller.rb

```diff
@@ -22,3 +23,4 @@ def authenticated?
     raise StandardError.new "No authentication is configured for ActiveStorage"
   end
 end
+# :nocov:
```

Comment:
Adding specs for this file seems inappropriate because

1. It's partially extracted from Rails itself
2. We are telling devs they must change or remove it before their uploads will work
3. We don't have any models etc. available to hand an ActiveStorage attachment from for testing

We have very little code so this being in code coverage was causing us to fail the coverage requirement.